go.mod

```diff
@@ -5,7 +5,7 @@ go 1.19
 replace (
 	github.com/Microsoft/hcsshim => github.com/Microsoft/hcsshim v0.8.22
 	github.com/Mirantis/cri-dockerd => github.com/k3s-io/cri-dockerd v0.3.2-0.20230123224936-bcd78c2d21d8 // k3s/release-1.26
-	github.com/cloudnativelabs/kube-router => github.com/k3s-io/kube-router v1.5.2-0.20221026101626-e01045262706
+	github.com/cloudnativelabs/kube-router => github.com/k3s-io/kube-router v1.5.1-0.20230223115732-df90811446a1
```

Comment:
I know this is just a golang module pseudo-version, but why does it look like the version is going backwards? Are we missing some upstream tags from our kube-router fork?

Reply:
This is related to the new version of kube-router. On the previous version we were using our kube-router branch with the dual-stack support. With this version I cloned the latest kube-router from upstream with dualstack support and this version is `v2.0.0` and go requires different names on the go module in case you are using `v2` and the upstream is using wrong names then I can't use the tag `v2.0.0` from our fork. (I can try to fix the go module file on the fork but I am not sure that it works).